addons/OPT/tracker/fn_addMarker.sqf

```diff
@@ -5,15 +5,15 @@
 *
 * Argumente:
 * 0: <ARRAY> _startPosition (Start Position des erstellten Marker)
-* 1: <Number> _prioritaet (0 = Realtime|40 = Normal,Zeitangabe für Aktuellsierung in der Tracking Funktion)
+* 1: <STRING> _prioritaet (Realtime|Normal,Zeitangabe für Aktuellsierung in der Tracking Funktion)
```

Comment:
Der Teil `Zeitangabe für Aktuellsierung in der Tracking Funktion` passt hier nicht länger.

Außerdem ist es praktisch für solche Strings, die nur aus einem Wort bestehen, alles klein zu schreiben. Dh. `realtime`  und `normal`